mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -21,6 +21,7 @@ import scala.annotation.varargs
 
 import org.apache.spark.annotation.Since
 import org.apache.spark.api.java.{JavaDoubleRDD, JavaRDD}
+import org.apache.spark.ml.stat.SummaryBuilderImpl._
```

Comment:
Design wise, can we minimize the places where mllib calls ml? and is it possible to expose this without reaching into the "Impl" class? it looks a little funny

Reply:
Just because current aggregator is an inner class `org.apache.spark.ml.stat.SummaryBuilderImpl.SummarizerBuffer`,
I guess I need to move it outside of `SummaryBuilderImpl`